src/main/java/net/minecraftforge/fml/common/network/simpleimpl/SimpleNetworkWrapper.java

```diff
@@ -237,6 +238,25 @@ private String generateName(ChannelPipeline pipeline, ChannelHandler handler)
     public void sendToAll(IMessage message)
     {
         channels.get(Side.SERVER).attr(FMLOutboundHandler.FML_MESSAGETARGET).set(FMLOutboundHandler.OutboundTarget.ALL);
+        channels.get(Side.SERVER).attr(FMLOutboundHandler.FML_MESSAGETARGETEXCEPT).set(null);
+        channels.get(Side.SERVER).writeAndFlush(message).addListener(ChannelFutureListener.FIRE_EXCEPTION_ON_FAILURE);
+    }
+
+    /**
+     * Send this message to everyone except one player.
+     * The {@link IMessageHandler} for this message type should be on the CLIENT side.
+     *
+     * This is particularly useful for player-caused events that require instant feedback. The excepted player will not
+     * receive the message, and is therefore expected to reliably predict the outcome of the event.
+     * Must be used with care when synchronizing game logic.
```

Comment:
The warning should be something like:
"Must not be used when synchronizing game logic, only instant visual or audio effects."
There is no case I can think of where it's safe to sync other kinds of things this way.
Even something as simple as rotating a block needs to go through the server and back to the client to confirm it actually happened.
It's important that this javadoc remain short so that the warning is what people read, or put the warning at the top.